addons/portal/data/mail_templates.xml

```diff
@@ -6,7 +6,9 @@
             <p>Dear <span t-esc="partner.name"/>,</p>
             <p>You have been invited to access the following document:</p>
             <br/>
-            <a t-attf-href="#{share_link}" style="background-color: #875A7B; padding: 10px; text-decoration: none; color: #fff; border-radius: 5px; font-size: 12px;"><strong>Open </strong><strong t-esc="record.display_name"/></a><br/>
+            <div style="margin-top: 1rem;">
```

Comment:
Maybe we can add this to the style of the `<a>`? Seems like the div isn't very useful anymore :)

Reply:
@thiryr 
I have checked in <a> but it is not working properly so I have added it in the <div> .
Thank you :)